.nox/design/allnoxtypes.entity.nox.yaml

```diff
@@ -8,8 +8,8 @@ keys:
 
   - name: Id
```

Comment:
Just an FYI: this was part of the SampleWebApp project that was deleted and it's not used anymore. Ricardo will delete the folder on main branch.